kalite/distributed/management/commands/kaserve.py

```diff
@@ -135,13 +140,66 @@ def handle(self, *args, **options):
 
         if options['startuplock']:
             os.unlink(options['startuplock'])
-        
+
         # Now call the proper command
         if not options["production"]:
+
+            if options['use_inotify']:
```

Comment:
If this represents a performance gain, why not include it in the "production" code path? In general, I think we should minimize the differences between the development environment and the production environment.